MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -1184,14 +1206,41 @@ private void PatchRelocation(MbbsModule module)
             }
         }
 
-        private void ProcessNightlyCleanup()
+        private void EnableModule(string moduleId)
         {
-            if (_globalCache.ContainsKey("SYSOPGLOBAL-CLEANUP"))
+            var _moduleId = moduleId;
+            var moduleChange = _moduleConfigurations.FirstOrDefault(m => m.ModuleIdentifier.Equals(_moduleId, StringComparison.InvariantCultureIgnoreCase));
```

Comment:
Will this persist in the `modules.json`? Ok if it doesn't -- but we'll probably want to make that something that happens eventually. Not in this PR.

Reply:
It does not now, if you restarted the BBS it would reset back to whatever was enabled/disabled in config ... for future pr